en/docs/setup/token-persistence.md

```diff
@@ -91,48 +92,39 @@ The following table describes what each of the above attributes means:
 </tbody>
 </table>
 
-The main difference between synchronous and asynchronous token
-persistence is that the OAuth2 component in the synchronous token
-persistence implementation waits for the access token to be persisted in
-the database before returning it to the client.
-
-### Recovery flow for token persistence
+## Recovery flow for token persistence
 
 This section explains the recovery flow triggered in WSO2 Identity
 Server for exceptional cases that may occur in a production environment
 caused by the client application mishandling the
 `         CON_APP_KEY        ` constraint that is explained below.
 
--   [CONN\_APP\_KEY
-    constraint](#TokenPersistence-CONN_APP_KEYconstraint)
--   [Asynchronous token
-    persistence](#TokenPersistence-Asynchronoustokenpersistence.1)
-    -   [The flow](#TokenPersistence-Theflow)
-    -   [The recovery flow](#TokenPersistence-Therecoveryflow)
--   [Synchronous token
-    persistence](#TokenPersistence-Synchronoustokenpersistence.1)
-    -   [The flow](#TokenPersistence-Theflow.1)
-    -   [The recovery flow](#TokenPersistence-Therecoveryflow.1)
+-   [CONN\_APP\_KEY constraint](#conn_app_key-constraint)
+-   [Asynchronous token persistence](#asynchronous-token-persistence_1)
+    -   [The flow](#the-flow_1)
+    -   [The recovery flow](#the-recovery-flow_1)
+-   [Synchronous token persistence](#synchronous-token-persistence_1)
+    -   [The flow](#the-flow_1)
+    -   [The recovery flow](#the-recovery-flow_1)
```

Comment:
Let's remove the TOC

Reply:
Fixed: 69387cab7c97dd931cffa7a97e11d3abc27ef908